src/objects/RecurringDonationStatusMapping__mdt.object

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Recurring Donation Status to State Mapping</description>
+    <fields>
+        <fullName>State__c</fullName>
+        <description>The state that the Recurring Donation Status will be mapped to.</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>Valid entries are “Active”, “Lapsed”, and “Closed”.</inlineHelpText>
+        <label>Recurring Donation State</label>
+        <required>false</required>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Active</fullName>
+                    <default>false</default>
+                    <label>Active</label>
+                </value>
+                <value>
+                    <fullName>Lapsed</fullName>
+                    <default>false</default>
+                    <label>Lapsed</label>
+                </value>
+                <value>
+                    <fullName>Closed</fullName>
+                    <default>false</default>
+                    <label>Closed</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Status__c</fullName>
+        <description>API name of the Recurring Donation Status that will be mapped to the state.</description>
+        <externalId>false</externalId>
+        <fieldManageability>SubscriberControlled</fieldManageability>
```

Comment:
I'm not so sure about this. I believe that this should be "DeveloperControlled" to match all other CMDT fields in our package. Even as DeveloperControlled field, Users are still able to modify the values in the field and create records. If I'm not mistaken, making this SubscriberControlled means we can never modify the attributes of the field.